extensions/pom.xml

```diff
@@ -19,7 +19,7 @@
     <description>Mule extensions that augment the core Mule functionality through the Mule Extension API</description>
 
     <modules>
-        <!-- <module>validation</module> -->
+         <module>validation</module>
```

Comment:
I don't see why that's a right first step. This API is not only meant for external extensions or connectors like devkit was. This is meant as the very foundation of future mule.

Thus, this is unrelated to the decision of keeping all components which are part of the distro together vs splitting each component to its own.